src/Symfony/Component/Stopwatch/StopwatchPeriod.php

```diff
@@ -73,4 +73,12 @@ public function getMemory()
     {
         return $this->memory;
     }
+
+    /**
+     * @return string
```

Comment:
this can be turned into a real type (or removed)

Reply:
Done :-)